region/brinstar/green.json

```diff
@@ -2585,7 +2585,20 @@
                       "enemy": "Beetom",
                       "type": "contact",
                       "hits": 2
-                    }}
+                    }},
+                    {"or": [
+                      "canPreciseCeilingClip",
+                      "canWalljump",
+                      "HiJump",
+                      "SpaceJump",
+                      "h_canJumpIntoIBJ",
+                      "canSpringBallJumpMidAir",
```

Comment:
IBJ doesn't work; its not fast enough and has a big chance of killing the beetom.
Springball jump would be tedious.  I think I'd rather leave that out and take the two extra hits.